<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IrregularPuzzle</class>
 <widget class="QDialog" name="IrregularPuzzle">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1500</width>
    <height>850</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1500</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>2000</width>
    <height>2000</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::ContextMenuPolicy::PreventContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>异形拼图游戏</string>
  </property>
  <widget class="QPushButton" name="btnRestart">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25pt &quot;Arial Unicode MS&quot;;</string>
   </property>
   <property name="text">
    <string>重开</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>1180</x>
     <y>545</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25pt &quot;Arial Unicode MS&quot;;</string>
   </property>
   <property name="text">
    <string>步数：</string>
   </property>
   <property name="textFormat">
    <enum>Qt::TextFormat::PlainText</enum>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdStep_2">
   <property name="geometry">
    <rect>
     <x>1270</x>
     <y>280</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="digitCount">
    <number>5</number>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBack">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>580</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25pt &quot;Arial Unicode MS&quot;;</string>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QLabel" name="time_labl">
   <property name="geometry">
    <rect>
     <x>1180</x>
     <y>280</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25pt &quot;Arial Unicode MS&quot;;</string>
   </property>
   <property name="text">
    <string>耗时：</string>
   </property>
   <property name="textFormat">
    <enum>Qt::TextFormat::PlainText</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="btnShowImage">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>500</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25pt &quot;Arial Unicode MS&quot;;</string>
   </property>
   <property name="text">
    <string>说明</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnUndo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>420</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25pt &quot;Arial Unicode MS&quot;;</string>
   </property>
   <property name="text">
    <string>撤销</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>20</y>
     <width>960</width>
     <height>760</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdStep">
   <property name="geometry">
    <rect>
     <x>1270</x>
     <y>545</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="digitCount">
    <number>5</number>
   </property>
  </widget>
  <widget class="QLabel" name="labelProgress">
   <property name="geometry">
    <rect>
     <x>1180</x>
     <y>400</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;Arial Unicode MS&quot;;</string>
   </property>
   <property name="text">
    <string>进度: 0/0</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>1180</x>
     <y>440</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="labelTitle">
   <property name="geometry">
    <rect>
     <x>1180</x>
     <y>40</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 24pt &quot;Arial Unicode MS&quot;; color: #2E8B57;</string>
   </property>
   <property name="text">
    <string>异形拼图</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelHint">
   <property name="geometry">
    <rect>
     <x>1180</x>
     <y>100</y>
     <width>261</width>
     <height>161</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Arial Unicode MS&quot;;</string>
   </property>
   <property name="text">
    <string>游戏说明：
1. 拖动拼图块到中央区域
2. 右键可旋转拼图块
3. 拼图块会自动吸附
4. 完成所有拼图块获胜</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelRotationHint">
   <property name="geometry">
    <rect>
     <x>1180</x>
     <y>620</y>
     <width>261</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Arial Unicode MS&quot;; color: #8B4513;</string>
   </property>
   <property name="text">
    <string>右键旋转拼图块
左键拖动拼图块</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnReturn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>660</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25pt &quot;Arial Unicode MS&quot;;</string>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>